src/dotnet-test-xunit/Program.cs

```diff
@@ -542,13 +542,7 @@ private void SendTestCompletedIfNecessary(bool designTime, bool list)
 
         private static ISourceInformationProvider GetSourceInformationProviderAdapater(XunitProjectAssembly assembly)
         {
-            var directoryPath = Path.GetDirectoryName(assembly.AssemblyFilename);
-            var assemblyName = Path.GetFileNameWithoutExtension(assembly.AssemblyFilename);
-            var pdbPath = Path.Combine(directoryPath, assemblyName + FileNameSuffixes.DotNet.ProgramDatabase);
-
-            return File.Exists(pdbPath)
-                ? new SourceInformationProviderAdapater(new SourceInformationProvider(pdbPath))
-                : null;
+            return new SourceInformationProviderAdapater(new SourceInformationProvider(assembly.AssemblyFilename));
```

Comment:
Don't we need a new version of Extensions.Testing.Abstractions to have the code that will be able to load the pdb from the assembly?